widgets/cdatetime/org.eclipse.nebula.widgets.cdatetime/src/org/eclipse/nebula/widgets/cdatetime/CDateTimeBuilder.java

```diff
@@ -131,6 +136,20 @@ public boolean hasHeader(int type) {
 		return false;
 	}
 
+	/**
+	 * @return Returns the minDate.
+	 */
+	public Calendar getMinDate() {
```

Comment:
Allows date to be altered. Is this your intention? If so then state that in the javadoc. If not then return a clone of date.